client/src/App.vue

```diff
@@ -124,16 +124,16 @@ import paymentModule from '@/modules/payment'
   },
   computed: {
     ...mapState([
-      'showNrSessionExpiryModal',
-      'rollbackOnExpire',
-      'checkInOnExpire'
+      'showNrSessionExpiryModal'
+      // 'rollbackOnExpire', // NOT USED
+      // 'checkInOnExpire' // NOT USED
```

Comment:
Although there was code to SET these state properties, there was no code to GET (ie, use) them.
I think this might be a useful feature in the future so I commented this out instead of deleting it outright.